cg/models/analysis.py

```diff
@@ -1,5 +1,14 @@
 from pydantic.v1 import BaseModel
 
+from cg.models.rnafusion.rnafusion import RnafusionQCMetrics
+from cg.models.tomte.tomte import TomteQCMetrics
+
 
 class AnalysisModel(BaseModel):
     """Metadata analysis model"""
+
+
+class NextflowAnalysis(AnalysisModel):
```

Comment:
not sure I understand why this is needed

Reply:
You mean the `AnalysisModel`? We're using it everywhere to typehint the common methods in the delivery report returning the analysis_metadata for example

Reply:
the nextflow subclass. But I don't understand this part very well so go ahead and don't mind me. I guess this will simplify collecting information for nextflow pipelines but I'll check in details another time with more time so I fully follow this part of the code